src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -104,7 +142,17 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
 
     const [editDialogOpen, setEditDialogOpen] = useState(undefined);
     const [editData, setEditData] = useState(undefined);
-
+    const dispatch = useDispatch();
+    const studyUpdatedForce = useSelector((state) => state.studyUpdated);
+    const [messageId, setMessageId] = useState('');
+    const [launchLoader, setLaunchLoader] = useState(false);
+    const updateType = useMemo(() => {
```

Comment:
outside the functionnal component :
`const UPDATE_TYPE = ['creatingInProgress',
            'updatingInProgress',
            'deletingInProgress', ];`
We could even put it in a separated file and import it in this component.

Reply:
it's used only in the scope of this component